pci/graphic_drivers/xf86-video-amdgpu/MHWDCONFIG

```diff
@@ -11,54 +11,24 @@ VENDORIDS="1002"
 DEVICEIDS=">/var/lib/mhwd/ids/pci/amdgpu.ids"
 
 # Dependencies
-DEPENDS="xf86-video-amdgpu"
-
+DEPENDS="xf86-video-amdgpu vulkan-radeon libva-mesa-driver libva-vdpau-driver mesa-vdpau"
+DEPENDS_64="lib32-vulkan-radeon lib32-libva-vdpau-driver lib32-mesa-vdpau"
 
+# Files
 XORGFILE="/etc/X11/mhwd.d/amdgpu.conf"
-MODULESFILE="/etc/modules-load.d/mhwd-amdgpu.conf"
-MODULESBLACKLIST="/etc/modprobe.d/mhwd-amdgpu.conf"
 
 post_install()
 {
-	MHWD_HEADING "${XORGFILE}"
-	MHWD_ADD_DEVICE_SECTION_FOR_EACH_BUSID "amdgpu" "${XORGFILE}" 1002
-	MHWD_ADD_DRI "${XORGFILE}"
-	MHWD_ADD_COMPOSITING "${XORGFILE}"
-	MHWD_ADD_BACKSPACE "${XORGFILE}"
-
-	# Enable next generation of DRI
-	sed -i "s/\tOption      \"DRI\"    \"true\"/\tOption      \"DRI3\"    \"1\"\n\tOption      \"DRI\"     \"3\"/g"  "${XORGFILE}"
-
-	MHWD_HEADING "${MODULESFILE}"
-	echo "amdgpu" >> "${MODULESFILE}"
-
-	MHWD_HEADING "${MODULESBLACKLIST}"
-	echo "blacklist radeon" >> "${MODULESBLACKLIST}"
-
-	# Unload modules if X is not running
-	if [ ! "$(pgrep X)" ];	then
-		rmmod -f radeon
-	fi
+    pacman -Sy "xorg-server>=1.19" --noconfirm --needed
```

Comment:
I'm Ok with this. I was just a bit frightened because some users came up with that they cant reach the desktop and manjaro boots only to a "blackscreen" and they could not fix this by mhwd -i pci video-<driver> and it turns out that xorg isnt installed for some reason.

In pacman, is there some way to mark a package as systemcritical? Since manjaro is a desktop-distro and wayland is also not ready to shine, it would make sense to markt it that way.

(But i'm OK with removing this, because it's a little dirty anyways 👍 )